### DISCUSSION TITLE
1-1 relationship cant access data from both side

### DISCUSSION THREAD

**Discussion by derech1e** [prisma/prisma#11826] [Q&A] [answered]
So I have the following prisma schema setup in my Nest Projekt:

```prisma
model users {
    id                  Int                 @id @default(autoincrement())
    uuid                String              @default(uuid())
    createdAt           DateTime            @default(now())
    updatedAt           DateTime            @updatedAt
    firstName           String
    authentication      authentications    @relation("Authentication", fields: [authenticationId], references: [id], onDelete: Cascade)
    authenticationId    Int
}

model authentications {
    id                          Int         @id @default(autoincrement())
    uuid                        String      @default(uuid())
    createdAt                   DateTime    @default(now())
    updatedAt                   DateTime    @updatedAt
    role                        Role        @default(USER)
    emailAddress                String      @unique
    password                    String
    isEmailConfirmed            Boolean     @default(false)
    currentHashedRefreshToken   String?
    user                        users?      @relation("Authentication")
}

enum Role {
  SUSPENSION    @map("SUSPENSION_ROLE")
  USER          @map("USER_ROLE")
  ADMIN         @map("ADMIN_ROLE")
}

```

Now I want to fetch the `users` from the Database. Therefor I call 
```javascript
await this._prismaService.users.findMany({});
```
... it returns undefined.

But when I execute the same statement with `authentications` instead, it works.
Like so:

```javascript
await this._prismaService.authentications.findMany({});
```

Can someone explain to me why this is happening?
I just want to check if the user exists with a given UUID.
Thank you for your help!

**Comment by derech1e** [comment, marked as answer]
I' ve solved the issue my self. The problem was located in the prisma middleware.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
